src/backend/command.cpp

```diff
@@ -99,17 +99,26 @@ namespace big
 		g_commands[command]->call(args, ctx);
 	}
 
-	std::vector<command*> command::get_suggestions(const std::string_view search, const int limit)
+	std::vector<command*> command::get_suggestions(std::string search, const int limit)
 	{
 		std::vector<command*> found_commands{};
 		for (auto& [k, command] : g_commands)
 		{
 			if (command->get_label().length() == 0)
 				continue;
 
-			if (command->get_name().contains(search))
+			std::string cmd_name     = command->get_name();
+			std::string cmd_label    = command->get_label();
+			std::string lower_search = search;
+
+			//transform all strings to lower case
+			std::transform(cmd_name.begin(), cmd_name.end(), cmd_name.begin(), tolower);
+			std::transform(cmd_label.begin(), cmd_label.end(), cmd_label.begin(), tolower);
+			std::transform(lower_search.begin(), lower_search.end(), lower_search.begin(), tolower);
```

Comment:
Search can probably be made lower case outside of the for loop unless I'm missing something.

Reply:
Yep, that was my mistake.
I fixed it in the last commit